gldas/download.py

```diff
@@ -130,8 +138,8 @@ def parse_args(args):
     parser.add_argument("-e", "--end", type=mkdate,
                         help=("Enddate. Either in format YYYY-MM-DD or YYYY-MM-DDTHH:MM."
                               "If not given then the current date is used."))
-    parser.add_argument("--product", choices=["GLDAS_Noah_v1_025"], default="GLDAS_Noah_v1_025",
-                        help='GLDAS product to download.')
+    parser.add_argument("--product", choices=["GLDAS_Noah_v20_025", "GLDAS_Noah_v21_025"], default="GLDAS_Noah_v20_025",
+                        help='GLDAS product to download. GLDAS_Noah_v1_025 data depreciated and replaced by newer versions.')
```

Comment:
This help text does not seem very helpful for a user wanting to download data. Could you replace it by `Dataset available from (startdate) (link to documentation)`?